packages/foundation/docs/themes.mdx

```diff
@@ -1,30 +1,232 @@
 ---
-name: Default theme
+name: Documentation
 menu: Theme
 ---
 
 import * as packageJson from '../package.json';
 import ThemeTokens from '../../../src/ThemeTokens';
 import { DefaultTheme } from '../src/themes';
 
-# Themes
+# Theme
 
-@uireact provides with a default theme that you can use:
+@uireact provides with a default theme in the foundation package which can be found here: [Default theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-theme.ts)
 
-## Light coloration
+For understading the concepts and theme creation you should continue reading this doc ⚡️
 
-[Light Theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-colors/light.ts)
+_10 mins read_
+
+## Set up ⚙️
+
+@uireact uses a theme file so components can be customizable for different clients / use cases. We believe that
+our structure provides with a lot of customization and flexibility without getting into too much freedom which causes
+UX design to be poor for the user experience.
+
+If you use the component `UiView` then it sets automatically out of the box the theme context, so you SHOULD wrap your entire app
+in a UiView, you can get more information about this component in the [UiView docs](https://uireact.io/packages-view-docs-view).
+
+TL;DR;
+
+```
+import { @UiView } from '@uireact/view';
+import { DefaultTheme, ThemeColor } from '@uireact/foundation';
+
+export const App = () => (
+  <UiView theme={DefaultTheme} selectedTheme={ThemeColor.light}>
+    ... {Your App} ...
+  </Uiview>
+);
+
+```
+
+If you don't want to use the view component then you will have to manually set the ThemeContext provided from the foundation package:
+
+```
+import { ThemeContext, DefaultTheme, ThemeColor } from '@uireact/foundation';
+
+export const App = () => (
+  <ThemeContext.Provider value={{ theme: DefaultTheme, selectedTheme: ThemeColor.light }}>
+    ... {Your App} ...
+  </ThemeContext.Provider>
+);
```

Comment:
All this section is trying to explain the usage but I'm not sure I understand the advantages or disadvantages of using `UiView` vs `ThemeContext` provider, I t may be worth clarifying a bit more the usage and when it makes sence to use each strategy